make/Build.gmk

```diff
@@ -47,7 +47,7 @@ $(foreach file,$(NATIVE_TEST_SOURCES),$(eval $(call BuildNativeLibrary,$(file),$
 $(BUILD_CLASSES_DIR):
 	$(MKDIR) -p $(BUILD_CLASSES_DIR)/org/openjdk/jextract/impl
 	$(FIXPATH) $(PANAMA_JAVA_HOME)/bin/javac \
-	    --release=22 \
+	    --release=23 \
```

Comment:
I suppose once we drop `--enable-preview` the `--release flag` can also be dropped, and will just default to the version of the JDK we are building with. This is fine as the same JDK is used to create the jextract runtime image.

Reply:
Fixed in [5380106](https://github.com/openjdk/jextract/pull/265/commits/53801062000157c03e8ed0d8066dc8efceca161f) but it seems like we are bound to have a conflict after you integrate anyway, I tried a few things and would always end up with a conflict.